packages/cli/src/lib/index.js

```diff
@@ -147,7 +153,32 @@ export const generateTemplate = (templateFilename, { name, root, ...rest }) => {
 
 export const readFile = (target) => fs.readFileSync(target)
 
-export const deleteFile = (target) => fs.unlinkSync(target)
+export const deleteFile = (file) => {
+  const extension = path.extname(file)
+  if (['.js', '.ts'].includes(extension)) {
+    const baseFile = getBaseFile(file)
+    const files = [baseFile + '.js', baseFile + '.ts']
+    files.forEach((f) => {
+      if (fs.existsSync(f)) {
+        fs.unlinkSync(f)
+      }
+    })
+  } else {
+    fs.unlinkSync(file)
+  }
+}
+
+const getBaseFile = (file) => file.replace(/\.\w*$/, '')
+
```

Comment:
Instead of adding `typescript` as a flag to `destroy`, it felt more ergonomic to me for it to just destroy whatever js/ts variation is found.